config/dependencies/fvt.yaml

```diff
@@ -53,7 +53,7 @@ spec:
             - /tmp/etcd.data
           # image: quay.io/coreos/etcd:v3.5.4
           # Tag -> registry.access.redhat.com/rhel7/etcd:3.2.32-34
-          image: registry.redhat.io/rhel7/etcd@sha256:d3495b263b103681f1b09a558be43c21989bfc269eb90f84c2609042cebdc626
+          image: registry.redhat.io/openshift4/ose-etcd@sha256:d3275cd886d13865937d225d8138db7f6b7bf59ac1a94d9fbe61e35286bee6ff
```

Comment:
is it the same image that kserve raw use?

Reply:
does kserve uses etcd?

Reply:
oh nvm, I thought it was oauth-proxy